db/schema.rb

```diff
@@ -757,6 +757,7 @@
     t.boolean "payment_overdue", default: false
     t.uuid "payable_group_id"
     t.bigint "negative_amount_cents", default: 0, null: false
+    t.bigint "progressive_billing_credit_amount_cents", default: 0, null: false
```

Comment:
I don't know why I have this change on a fresh DB, maybe @nudded can help?

Reply:
That should not be possible 🤔 i will check